ygen/testdata/structs/openconfig-withlist-opstate.formatted-txt

```diff
@@ -185,8 +185,8 @@ func (t *Model) RenameSingleKey(oldK, newK string) error {
 
 // Model_MultiKey represents the /openconfig-withlist/model/b/multi-key YANG schema element.
 type Model_MultiKey struct {
-	Key1	*uint32	`path:"state/key1|key1" module:"openconfig-withlist"`
-	Key2	*uint64	`path:"state/key2|key2" module:"openconfig-withlist"`
+	Key1	*uint32	`path:"state/key1" module:"openconfig-withlist"`
```

Comment:
This doesn't look right... At this point you're saying that we don't serialise this value into the direct child which would be invalid for the schema - i.e., for schema

```yang
list foo
   config false;
   key "a";

   leaf a {
       type leafref {
          path "../state/a";
       }
   }
   
   container state {
       leaf a { type string; }
   }
}
```

we need to produce:

```
/foo/a
/foo/state/a
```

In this case, it seems like you removed the ability to produce /foo/a - which is invalid, since now `key "a"` is nil under the list.

Reply:
Yes this change does remove `/foo/a` if the leafref points to `"../config/a"`, but I believe in your example it should keep it since it points to the node under state.

enum-module.yang has this snippet:
```
  container a-lists {
    list a-list {
      config false;
      key "value";

      leaf value {
        type leafref {
          path "../state/value";
        }
      }

      container state {
        config false;
        uses common-value;
      }
    }
  }
```

enum-module.formatted-txt:
```
// AList represents the /enum-module/a-lists/a-list YANG schema element.
type AList struct {
    Value   AList_Value_Union   `path:"state/value|value" module:"enum-module"`
}
```